### ISSUE TITLE
Check if can filter out NCT Clinical Trial records that aren't COVID

### ISSUE THREAD

**Issue by flaneuse** [SuLab/outbreak.info-resources#80]
See "NCT02984761" -- mentions COVID-19 in text, but isn't actually a COVID-19 trial

**Comment by andrewsu** [comment]
one hackish option to consider would be to maintain whitelists and blacklists for each automated harvester...

**Comment by flaneuse** [comment]
Yeah; one thing I need to check is if the [NCT table](https://clinicaltrials.gov/ct2/results?cond=COVID-19) contains these sorts of trials. The API gives more results than their webpage, and this might be a reason why.

Unfortunately, it doesn't seem possible to do fielded queries in their API, so every search looks through every field.
